src/common/gdicmn.cpp

```diff
@@ -370,7 +370,13 @@ void wxColourDatabase::Initialize()
         {wxT("WHEAT"), 216, 216, 191},
         {wxT("WHITE"), 255, 255, 255},
         {wxT("YELLOW"), 255, 255, 0},
-        {wxT("YELLOW GREEN"), 153, 204, 50}
+        {wxT("YELLOW GREEN"), 153, 204, 50},
+        {wxT("wxNAVY"), 0, 0, 128},
```

Comment:
This doen't work. All characters need to be capitalized to find it. And why prefix with `wx`?

Reply:
@MaartenBent 
Because `NAVY` is already exists? But `wxNAVY` does not.
And the ::Find() call calls `MakeUpper()` already, so I don't think it matters.

Reply:
That is why it will not work. `Find()` will search for `WXNAVY`, but `wxNAVY` is in the db.

Reply:
Still broken. `wxNAVY` is in the database, but you search for `WXNAVY`. And you need to add `{wxT("OLIVE"), 128, 128, 0}` to the database. And `wxDARK GREY` is the same as `GREY`.
And why use `WX` prefix for names that are not in the database, like `TEAL` and `LIGHT GREEN`?
And maybe add them to the database in alphabetical order.